src/slice/mod.rs

```diff
@@ -73,8 +73,8 @@ pub trait ParallelSlice<T: Sync> {
     ///
     /// ```
     /// use rayon::prelude::*;
-    /// let windows: Vec<_> = [1, 2, 3, 4].par_chunks(2).collect();
-    /// assert_eq!(vec![[1, 2], [3, 4]], windows);
+    /// let chunks: Vec<_> = [1, 2, 3, 4, 5].par_chunks(2).collect();
+    /// assert_eq!(chunks, vec![&[1, 2][..], &[3, 4], &[5]]);
```

Comment:
We don't need the trailing `[..]` here, right?

Reply:
It needs to be forced as a slice instead of array, or else the type of `[5]` won't match.

I think I could do this like `let expected: Vec<&[_]> = ...` if that's clearer.

Reply:
Ah yep. I personally think the way it is now is fine to read, I was just nit-picking for the sake of the review.